docs/usage_and_configuration/file_collection_and_discovery.md

```diff
@@ -24,7 +24,7 @@ as .pyi, and whether string normalization was omitted.
 
 To override the location of these files on all systems, set the environment variable
 `BLACK_CACHE_DIR` to the preferred location. Alternatively on macOS and Linux, set
-`XDG_CACHE_HOME` to you your preferred location. For example, if you want to put the
+`XDG_CACHE_HOME` to your preferred location. For example, if you want to put the
 cache in the directory you're running _Black_ from, set `BLACK_CACHE_DIR=.cache/black`.
```

Comment:
```suggestion
`XDG_CACHE_HOME` to your preferred location. For example, if you want to put the
cache in the directory you're running _Black_ from, set `BLACK_CACHE_DIR=.cache/black`.
```
Prettier

Reply:
```suggestion
`XDG_CACHE_HOME` to your preferred location. For example, if you want to put the cache
in the directory you're running _Black_ from, set `BLACK_CACHE_DIR=.cache/black`.
```
I'm.. 99 % sure your suggestion is identical to the origin, did you mean the above?

Reply:
Oh of course it shows the diff. So yeah, I think the `_Black_` on the next line won't fit the previous so this should satisfy Prettier.